idea_town/frontend/static-src/app/templates/experiment-row-view.js

```diff
@@ -1,10 +1,8 @@
 export default `
-  <li class="idea-card" data-hook="show-detail"
-    // toggle a class for styling purposes:
-    class="{{#isInstalled}} active {{/isInstalled}}">
-    <img width="200" height="200" src="{{thumbnail}}">
+  <li data-hook="show-detail" class="idea-card {{#isInstalled}} active {{/isInstalled}}">
+    <div class="idea-preview-image" style="background: url({{thumbnail}}) no-repeat"></div>
     <h2>{{title}}</h2>
-    <p>Status: {{^isInstalled}} Not {{/isInstalled}} Installed</p>
+    <p>{{description}}</p>
```

Comment:
One thing I'm not doing in this patch is ellipsizing any description overflow. I'm thinking the better answer might be to have separate `description` and `tagline` properties in the API, or something.